\hypertarget{md_Templates_README_autotoc_md37}{}\doxysubsubsection{Escuela de Ingeniería Eléctrica}\label{md_Templates_README_autotoc_md37}
\hypertarget{md_Templates_README_autotoc_md38}{}\doxysubsubsection{Faculta de Ingeniería}\label{md_Templates_README_autotoc_md38}
\hypertarget{md_Templates_README_autotoc_md39}{}\doxysubsubsection{IE0217-\/\+Estructuras de Abstractas de Datos y Algoritmos}\label{md_Templates_README_autotoc_md39}
\hypertarget{md_Templates_README_autotoc_md40}{}\doxyparagraph{Profesor\+: Esteban Badilla}\label{md_Templates_README_autotoc_md40}
\hypertarget{md_Templates_README_autotoc_md41}{}\doxyparagraph{Estudiante\+: Josué Salmerón Córdoba}\label{md_Templates_README_autotoc_md41}
\hypertarget{md_Templates_README_autotoc_md42}{}\doxyparagraph{I ciclo 2023}\label{md_Templates_README_autotoc_md42}
\hypertarget{md_Templates_README_autotoc_md43}{}\doxysubsubsection{Tarea 2}\label{md_Templates_README_autotoc_md43}
\hypertarget{md_Templates_README_autotoc_md44}{}\doxysection{Resumen}\label{md_Templates_README_autotoc_md44}
\begin{quote}
En esta segunda parte de la tarea2 se comentó el header que no era necesario para correr el código. Al parecer fueron 2\+: en este caso {\ttfamily algorithm} y {\ttfamily vector}, pero resulta que vscode se quejaba ya que lo resaltaba en rojo pero luego omitía ese error el script corría sin ningún problema, en todo caso mejor se dejó tal como estaba. Después se realizaron los respectivos comentarios de clase \mbox{\hyperlink{classStack}{Stack}} y sus métodos y atributos. También se declaró una nueva instancia donde se pueden ver sus valores a través de la consola. Y por último se implementó una instrucción que lanza la excepción {\ttfamily out\+\_\+of\+\_\+range}, no obstante tanto su declaración como el {\ttfamily catch} se dejaron en forma de comentario. \end{quote}
\hypertarget{md_Templates_README_autotoc_md45}{}\doxysection{Desarrollo de la excepción creada.}\label{md_Templates_README_autotoc_md45}

\begin{DoxyItemize}
\item Básicamente lo que se hizo fue crear la instancia {\itshape t} y vaciar la pila sin que tuviera ningún elemento, lo cual hace un llamado a la excepción\+: {\ttfamily out\+\_\+of\+\_\+range}, pero como se tiene un {\itshape catch}, entonces no hay un segmetation fault para el programa, lo que si sucede es que el script termina de manera inmediata. 
\end{DoxyItemize}\hypertarget{md_Templates_README_autotoc_md46}{}\doxysection{Comandos para correr la tarea}\label{md_Templates_README_autotoc_md46}

\begin{DoxyItemize}
\item Basta con abrir un terminal dentro de la carpeta {\ttfamily Templates}y escribir el siguiente comando\+: 
\begin{DoxyCode}{0}
\DoxyCodeLine{make}

\end{DoxyCode}
 
\end{DoxyItemize}